websockify/websockifyserver.py

```diff
@@ -671,11 +687,17 @@ def top_new_client(self, startsock, address):
         try:
             try:
                 client = self.do_handshake(startsock, address)
-            except self.EClose:
+            except self.EClose as e:
                 _, exc, _ = sys.exc_info()
                 # Connection was not a WebSockets connection
                 if exc.args[0]:
-                    self.msg("%s: %s" % (address[0], exc.args[0]))
+                    orig_client = getattr(e, 'orig_client', None)
```

Comment:
I don't think we should start piggy-backing extra information on the exception object. Better to add this information to the connection object somewhere.